src/orange/orangeinp/ConvexRegion.hh

```diff
@@ -230,6 +230,60 @@ class InfWedge final : public ConvexRegionInterface
     Turn interior_;
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * A general parallelepiped centered on the origin.
+ *
+ * A parallelepiped is a shape having 3 pairs of parallel faces out of
+ * which one is parallel with the XY plane (Z faces). All faces are
+ * parallelograms in the general case. The Z faces have 2 edges parallel
+ * with the X-axis.
+ *
+ * The shape has the center in the origin and it is defined by:
+ *
+ *   - `dX, dY, dZ:` half-lengths of the projections of the edges on X, Y
+ *     and Z. The lower Z face is positioned at `-dZ`, while the upper at
+ * `+dZ`.
+ *   - `alpha:` angle between the segment defined by the centers of the
+ *     X-parallel edges and Y axis `[-90,90]` in degrees;
```

Comment:
```suggestion
 *     X-parallel edges and Y axis, up to a quarter turn (i.e. 90 degrees);
```
Is it supposed to be valid to have a -90 angle? The validation in the constructor prevents that.

Reply:
An alpha angle of +/-90deg would make Y become X, which would degenerate that face and invalidate the shape.  So I have updated this comment and kept the validation.
I have also included the negative angles, which I had originally excluded.